d20potz.py

```diff
@@ -355,29 +341,138 @@ async def cardsCommand(update: Update, context: ContextTypes.DEFAULT_TYPE):
             return
 
         card_name = params[3]
-        player_cards = list(filter(lambda card: card.find(card_name) != -1, CARDS[player_name]))
+        player_cards = list(filter(lambda card: card.find(
+            card_name) != -1, CARDS[player_name]))
         if len(player_cards) == 0:
             await context.bot.send_message(
                 chat_id=chat_id,
-                text="{} Could not find in {}".format(card_name, CARDS[player_name]),
+                text="{} Could not find in {}".format(
+                    card_name, CARDS[player_name]),
             )
         else:
             await sendCards(player_name, player_cards, chat_id, context)
         return
     elif sub_command == "draw":
-        #TODO
+        if len(params) < 3:
+            await context.bot.send_message(
+                chat_id=chat_id, text="Usage: /cards <player> draw <card name>"
+            )
+            return
+
+        card_name = params[3]
+        player_cards = list(filter(lambda card: card.find(
+            card_name) != -1, CARDS[player_name]))
+        if len(player_cards) == 0:
+            await context.bot.send_message(
+                chat_id=chat_id,
+                text="{} Could not find in {}".format(
+                    card_name, CARDS[player_name]),
+            )
+        setPlayerCardStatus(chat_id, player_name,
+                            player_cards[0], False)
```

Comment:
Plz use named params if there are more than ~3 of them, or meaning is not clear.

```suggestion
                            player_cards[0], flipped=False)
```